### DISCUSSION TITLE
Floating toolbar open when user text selection.

### DISCUSSION THREAD

**Discussion by mateen777** [taiga-family/taiga-ui#9991] [Q&A] [answered]
@splincode can we open floating toolbar when user select text like how @tiptap/bubble-menu ? if not so how can i use bubble-menu by passing html element for tippyjs in editor?

**Comment by splincode** [comment]
https://taiga-family.github.io/editor/toolbar/floating

**Reply by mateen777** [reply]
![Screenshot (226)](https://github.com/user-attachments/assets/3f537092-39d3-430d-87ed-5bb83e1dff2a)

in taiga editor whithout selection of text or content also it is showing but i'm looking like above image.

**Reply by splincode** [reply, marked as answer]
`but i'm looking like above image.` There is no such support in the editor.
